src/web3_extentions/tx_utils.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from typing import Optional
+
+from eth_account.signers.local import LocalAccount
+from web3.exceptions import ContractLogicError
+from web3.module import Module
+from web3.types import TxParams
+
+from src.metrics.prometheus.basic import TX_SEND, TX_FAILURE
+
+
+logger = logging.getLogger(__name__)
+
+
+class TransactionUtils(Module):
+    @staticmethod
+    def check_transaction(transaction, from_address: str) -> bool:
+        """
+        Returns:
+        True - transaction succeed.
+        False - transaction reverted.
+        """
+        logger.info({"msg": "Check transaction locally.", "value": transaction.args})
```

Comment:
not sure that it's "locally" since it makes a static call to the node. just a thought, not need to fix it if you mind